uefi/src/proto/network/ip4config2.rs

```diff
@@ -0,0 +1,137 @@
+#![cfg(target_os = "uefi")]
+
+extern crate alloc;
```

Comment:
```suggestion
```

You can gate everything using "cfg(feature ="alloc")" as done in other places of the crate